jujugui/static/gui/src/test/test_sandbox_go.js

```diff
@@ -20,20 +20,18 @@ with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 (function() {
 
-  describe('sandbox.GoJujuAPI', function() {
+  describe('sandbox.JujuAPI', function() {
```

Comment:
I'm assuming this change is because there's no longer anything *but* the Go Juju API?